website/index.html

```diff
@@ -104,7 +104,7 @@
       function getStylesheet() {
             const currentTime = new Date().getHours();
```

Comment:
Can we test this across browsers to ensure const is supported? Else just use var